HW4/kondnataliia/HW4_1.py

```diff
@@ -0,0 +1,15 @@
+number = int(input("Enter your number: "))
+factorial = 1
+if number > 0:
+    for element in range(1, number+1):
+        factorial = factorial * element
+    print(f"The factorial of six is {factorial}.")
+
+# or
+
+number = int(input("Enter your number: "))
+factorial = 1
+while number > 1:
+    factorial = factorial * number
+    number -= 1
+print(f"The factorial of six is {factorial}.")
```

Comment:
incorrect message, why outputs a factorial of six?

Reply:
The same reason(